src/main/resources/com/adobe/epubcheck/messages/MessageBundle.properties

```diff
@@ -63,7 +63,6 @@ CSS_024=CSS class Selector is not used.
 CSS_024_SUG=Remove unused CSS selectors.
 CSS_025=CSS class Selector could not be found.
 CSS_025_SUG=Check for typos or define a class selector to document the use of the class.
-CSS_027=CSS selector specifies absolute position.
```

Comment:
I'm wondering if we should "shift" the other messages, i.e. s/CSS_028/CSS_027.
Removing messages is quite rare, I don't think this happened before.

These message codes always felt a bit odd –at best arbitrary–; maybe we can leave it like that for now, and revisit this when we refactor after v4.2.0.

Reply:
I wondered about that, but there are already a few gaps and I didn't want to introduce any unrelated problems. (-014, -018, and -026 are missing from the CSS messages, for example, and I see similar holes in HTM, OPF, etc.) I'd vote for leaving until we come up with a more logical system.